app/views/shared/_summary_card_component.html.erb

```diff
@@ -0,0 +1,41 @@
+<%
+  id ||= nil
+  data_attributes ||= {}
+  summary_card_actions ||= []
+  rows ||=[]
+%>
+<%= tag.div class: "app-c-summary-card-component", id: id, data: data_attributes do %>
+  <div class="govuk-summary-card">
+    <div class="govuk-summary-card__title-wrapper">
+      <h2 class="govuk-summary-card__title"><%= title %></h2>
+      <ul class="govuk-summary-card__actions">
+        <% summary_card_actions.each do |action| %>
+          <li class="govuk-summary-card__action">
+            <%= link_to sanitize(action[:label] + tag.span(" #{title}", class: "govuk-visually-hidden")), action[:href], class: "govuk-link govuk-link--no-visited-state #{"gem-link--destructive govuk-!-font-weight-bold" if action[:destructive]}".strip %>
```

Comment:
Very small point: there's no need to add a space in the visually hidden text here (and elsewhere) since the `govuk-visually-hidden` CSS class does that. Not a big deal though. 

I do wonder how useful the hidden text is though for a unique action like "Edit first publication date". If we consider it from the perspective of a screen reader user it might even add confusion as the screen reader would announce "Edit first publication date about this manual". It might be worth considering removing the hidden text altogether depending on the contexts in which the component is used. 

This would be even more the case for the "Reorder sections" action, which would read as "Reorder sections sections". In that case, and if we keep the hidden text, could we just display "Reorder" and "Add"?

Reply:
Fair points. This component was copied from Whitehall publisher, and I think I'd prefer to make the simpler change of updating the label text than having to add support for conditionally adding the visually hidden label.

I've spoken to our product owner and XD, and they're happy to update the section actions to "Reorder" and "Add", so we'll implement that change.

Reply:
Yes, I was getting a bit pedantic here - sorry! There's a couple of issues that have come up on Whitehall in the same sort of area so we may make some updates on that some time. But that's all good for me! 👍